README.md

```diff
@@ -7,145 +7,68 @@
-  - Tokens and NFTs ready for use, with indexed transaction history, and easy extension/customization
-- On-chain/off-chain orchestration
-  - Enterprise data flows backed by blockchain, with secure off-chain transfer of private docs+data
-  - Pluggable private data exchange / messaging (inc. [HTTPS + Mutual TLS](https://github.com/hyperledger/firefly-dataexchange-https))
-- Identity, data format, and interface distribution
-  - Broadcast data schema, proven identity, and on-chain logic integration APIs across the network
-  - Pluggable data distribution network with batch optimization (inc. [IPFS](https://ipfs.io/))
-  - *Pluggable DIDs for identity and multi-protocol on-chain interface definition are work in progress*
-- Microservice architecture, optimized for docker deployment
-  - Fully pluggable architecture, embracing multiple runtime technologies (Go, Node.js, Java etc.)
-- Built by developers for developers
-  - Ready to go in minutes, with a CLI, built-in UI explorer, OpenAPI spec, and samples
-- Data operations at the boundary of your data center
-  - Fast database cache + audit of all data flowing out of your enterprise, to the network
+## Start using Hyperledger FireFly
 
-## Quick Start Guide
+The best place to learn about FireFly is in the [documentation](https://hyperledger.github.io/firefly).
 
-Follow the [get started](https://hyperledger.github.io/firefly/gettingstarted/gettingstarted.html) guide in the doc, and your
-local developer environment will be up in minutes.
+There you will find our [Getting Started Guide](https://hyperledger.github.io/firefly/gettingstarted/gettingstarted.html),
+which will get you a running FireFly network of Supernodes on your local machine in a few minutes.
 
-You'll have your own private multi-party system, comprising a blockchain (Ethereum/Fabric) with API/Event connectors, a Private Data Exchange, an IPFS data sharing network, and ERC-1155 Token/NFT implementations.
+Your development environment will come with:
 
-All with the Hyperledger FireFly Explorer UI of course, and a [samples to get you building fast](https://github.com/hyperledger/firefly-samples).
+FireFly CLI                   |  FireFly Explorer UI                | FireFly Sandbox  |
+:----------------------------:|:-----------------------------------:|:----------------:|
+![](./images/firefly_cli.png) | ![](./images/firefly_explorer.png) | ![](./images/firefly_sandbox.png) |
 
-![FireFly Explorer](images/firefly_explorer.png)
+## Engage with the community
 
-## API Reference
+- [Join us on Discord](https://discord.gg/Fy7MJuqw86)
 
-All the Hyperledger FireFly APIs are self-documenting via Swagger, and you can just open them up on `/api` on your running FireFly.
+## Technical architecture
 
-Or you can check out the [latest API here](https://hyperledger.github.io/firefly/swagger/swagger.html).
+Hyperledger FireFly is a pluggable microservices architecture. Everything is pluggable, from the the Blockchain technology,
```

Comment:
Perhaps "Hyperledger FireFly has a pluggable microservices architecture"? I don't think of FireFly _being_ architecture, as much as it is _having_ a really good architecture.